ingestion/src/metadata/ingestion/api/topology_runner.py

```diff
@@ -283,7 +264,7 @@ def _(
             )
 
         # We want to keep the full payload in the context
-        self.update_context(stage=stage, entity=right)
+        self.update_context(stage=stage, entity_name=right)
```

Comment:
in this method we are missing removing the GET I believe

Reply:
and same as with the lineage, not sure why we are passing the full `right` to the entity_name

Reply:
Modified the call